src/porepy/grids/boundary_grid.py

```diff
@@ -77,3 +86,28 @@ def projection(self):
 
         """
         return self._proj
+
+    def __repr__(self) -> str:
+        """Get a string representation of the boundary grid.
+
+        Returns:
+            A string representation of the boundary grid.
+
+        """
+        s = f"Boundary grid of dimension {self.dim}\n"
+        s += f"ID of parent grid: {self._parent.id}\n"
+        s += f"{self.num_cells} cells.\n"
```

Comment:
suggestion: Boundary grid of dimension self.dim containing self.num_cells cells. ID of parent grid self._parent.id and dimension of the projection...